submissions/akulahere/DOM model/src/scripts/main.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const galleryList = document.querySelector('.gallery_list');
+const bigImage = document.querySelector('.gallery_large-img');
+const title = document.querySelector('figcaption');
+
+title.textContent = "Day and night"
+
+galleryList.addEventListener('click', (event) => {
+  event.preventDefault();
+  bigImage.src = event.target.closest('.list-item_link').href;
```

Comment:
What happens if I click the space between menu items?

Reply:
I refactored this fragment to avoid the bug. Now it works only when you click exactly on the element.